packages/components/src/inputs/src/InputGroup.module.css

```diff
@@ -0,0 +1,92 @@
+.hop-InputGroup  {
+    --input-group-background: var(--hop-neutral-surface);
```

Comment:
css variable naming scheme is wrong here

Reply:
are these ones supposed to be --hop-InputGroup?
I was going to say we could use internal variables here too, but I guess if it's not reset then it's not needed... Maybe the others have an opinion on this because I'm confused with that part too.